lib/webrtc/call.js

```diff
@@ -260,6 +260,60 @@ MatrixCall.prototype.hangup = function(reason, suppressEvent) {
 };
 
 /**
+ * Set whether the local video preview should be muted or not.
+ * @param {boolean} muted True to mute the local video.
+ */
+MatrixCall.prototype.setLocalVideoMuted = function(muted) {
+    if (!this.localAVStream) {
+        return;
+    }
+    setTracksEnabled(this.localAVStream.getVideoTracks(), !muted);
+};
+
+/**
+ * Check if local video is muted.
+ *
+ * If there are multiple video tracks, <i>all</i> of the tracks need to be muted
+ * for this to return true. This means if there are no video tracks, this will
+ * return true.
+ * @return {Boolean} True if the local preview video is muted, else false
+ * (including if the call is not set up yet).
+ */
+MatrixCall.prototype.isLocalVideoMuted = function() {
```

Comment:
This could be called, 'camera' to match 'Microphone'? I guess it could be a screen capture source but 'microphone' could probably be other things too.